src/types.ts

```diff
@@ -20,6 +20,10 @@ type GetCacheKeyAsync = (args: {
 
 export type GetCacheKey = GetCacheKeySync | GetCacheKeyAsync;
 
+export type Chunk = string | Buffer;
+
+export type PurgeFunction = (tag: string) => void;
```

Comment:
Don't forget to export the `PurgeFunction` type from the library entrypoint (`index.ts`)

Reply:
I think we should document this type in [the readme](https://github.com/etienne-martin/express-aggressive-cache#typescript) too.

Reply:
I was thinking of doing this when it does something useful (future PR).